controllers/propagator/propagation.go

```diff
@@ -1229,30 +1096,24 @@ func isConfigurationPolicy(policyT *policiesv1.PolicyTemplate) bool {
 	return jsonDef != nil && jsonDef["kind"] == "ConfigurationPolicy"
 }
 
-func (r *PolicyReconciler) isPolicySetSubject(instance *policiesv1.Policy, subject policiesv1.Subject) bool {
-	log := log.WithValues("policyName", instance.GetName(), "policyNamespace", instance.GetNamespace())
-
-	if subject.APIGroup == policiesv1.SchemeGroupVersion.Group &&
-		subject.Kind == policiesv1.PolicySetKind {
-		policySetNamespacedName := types.NamespacedName{
-			Name:      subject.Name,
-			Namespace: instance.GetNamespace(),
-		}
+func (r *PolicyReconciler) PolicySetHasPolicy(policySetName, policyName, namespace string) bool {
```

Comment:
function name? maybe hasPolicySetInPolicSet hm... weird verb is not first but not a big deal

Reply:
Maybe `IsPolicyInPolicySet`?

Reply:
Good!